Statusseite: ImageName, Mesh-MAC, Tunnel-MAC und Sitecode.

Fuer Gluon 2025.1 neu geschrieben.

Die Zeile "Gluon Version" ist entfallen: sie steckt bei Upstream in der
Firmware-Zeile als "release (base)".

Target/Subtarget zeigt Upstream ebenfalls schon, aber als anhaengendes
<dd> im Firmware-Block und damit ohne eigene Beschriftung. Dieser Patch
nimmt es dort heraus und setzt es als eigene Zeile hinter das Modell,
vor ImageName - dort, wo es in der 2023.2-Fassung stand und wo man es
sucht. Das Format "target (subtarget)" bleibt.

Die Tunnel-MAC haengt jetzt als eigener if-Block hinter dem
vpn_pubkey-Block, statt dessen Verschachtelung umzubauen. 2025.1 hat den
mesh_vpn-Bereich um den public key erweitert; ein eigener Block ist
robuster gegen weitere Aenderungen dort.

--- a/package/gluon-status-page/files/lib/gluon/status-page/view/status-page.html
+++ b/package/gluon-status-page/files/lib/gluon/status-page/view/status-page.html
@@ -165,10 +165,20 @@
 						</a></dd>
 					<%- end %>
 					<dt><%:Model%></dt><dd><%| nodeinfo.hardware.model %> (<%| nodeinfo.hardware.nproc %> CPUs)</dd>
+					<% if nodeinfo.software.firmware.target then -%>
+						<dt><%:Target/Subtarget%></dt><dd><%| nodeinfo.software.firmware.target %> (<%| nodeinfo.software.firmware.subtarget %>)</dd>
+					<%- end %>
+					<% if nodeinfo.software.firmware.image_name then -%>
+						<dt><%:ImageName%></dt><dd><%| nodeinfo.software.firmware.image_name %></dd>
+					<%- end %>
 					<dt><%:Primary MAC address%></dt><dd><%| nodeinfo.network.mac %></dd>
+					<% local mesh_if = nodeinfo.network.mesh and nodeinfo.network.mesh.bat0
+							and nodeinfo.network.mesh.bat0.interfaces %>
+					<% if mesh_if and mesh_if.other and mesh_if.other[1] then -%>
+						<dt><%:Mesh MAC-Adresse%></dt><dd><%| mesh_if.other[1] %></dd>
+					<%- end %>
 					<dt><%:IP address%></dt><dd><%= pcdata(table.concat(sorted(nodeinfo.network.addresses), '\n')):gsub('\n', '<br>') %></dd>
 					<dt><%:Firmware%></dt><dd><%| nodeinfo.software.firmware.release %> (<%| nodeinfo.software.firmware.base %>)</dd>
-					<dd><%| nodeinfo.software.firmware.target %> (<%| nodeinfo.software.firmware.subtarget %>)</dd>
 					<% if nodeinfo.network.mesh_vpn then -%>
 						<dt><%:Mesh VPN%></dt>
 						<dd>
@@ -179,6 +189,12 @@
 							<% if vpn_pubkey then -%>
 							<br><%| vpn_pubkey %> (public key)
 							<%- end %>
+							<% local tun = nodeinfo.network.mesh and nodeinfo.network.mesh.bat0
+									and nodeinfo.network.mesh.bat0.interfaces
+									and nodeinfo.network.mesh.bat0.interfaces.tunnel %>
+							<% if tun and tun[1] then -%>
+								<br>mac: <%| tun[1] %>
+							<%- end %>
 						</dd>
 						<% if nodeinfo.network.mesh_vpn.bandwidth_limit.enabled then -%>
 						<dt><%:Bandwidth limit%></dt>
@@ -192,6 +208,9 @@
 						</dd>
 						<%- end %>
 					<%- end %>
+					<% if nodeinfo.system.site_code then -%>
+						<dt><%:Sitecode%></dt><dd><%| nodeinfo.system.site_code %></dd>
+					<%- end %>
 					<dt><%:Site%></dt><dd><%| site.site_name() %></dd>
 					<% if nodeinfo.system.domain_code then -%>
 						<dt><%:Domain%></dt>
